### PR TITLE
Add comments to publish YAML fields

### PR DESCRIPTION
**Pull request by MDrakos**
<table>
<td>
  <a href="https://activestatef.atlassian.net/browse/DX-2496" title="DX-2496" target="_blank">BugDX-2496</a>  Mandatory fields have no examples in the YAML
  </td></table>
  <br />
 

These comments are my best guess at what we want to inject into the yaml. Please provide any suggestions to improve clarity.

### CODE REVIEW

**Comment by Naatan** [comment]
@rawktron the story does not provide all the necessary copy, so please review the copy on this PR.

**Review by Naatan — CHANGES_REQUESTED** [review]
Looks good from a technical perspective, but tagged Pete to review the copy.

Imo some of the descriptions are pointless noise, eg. on dependencies and features. A yaml comment wouldn't provide enough space to actually explain these.

Perhaps we can link to the documentation for some of this?

**Review by Naatan — APPROVED** [review]
Woops, meant to approve. But wait for Pete's approval before merging please.

**Review by rawktron — APPROVED** [review]

**Pull request merged**
